io.github.ripose.Memento.yml

```diff
@@ -0,0 +1,577 @@
+app-id: io.github.ripose.Memento
```

Comment:
this is incorrect. The user name for the repo is `ripose-jp` so it should be `io.github.ripose_jp.Memento`

Reply:
like the homepage.

Reply:
Fixed